internal/util/get/get.go

```diff
@@ -48,14 +49,22 @@ type Command struct {
 
 // Run runs the Command.
 func (c Command) Run() error {
+	revertFunc, err := c.updateParentKptfile()
+	if err != nil {
+		return err
+	}
+
 	r := &git.RepoSpec{OrgRepo: c.Repo, Path: c.Directory, Ref: c.Ref}
-	err := (&fetch.Command{
+	err = (&fetch.Command{
 		RepoSpec:    r,
 		Destination: c.Destination,
 		Name:        c.Name,
 		Clean:       c.Clean,
 	}).Run()
 	if err != nil {
+		// Ignore the error here. If this happens, it just means that
+		// we weren't able to roll back the change to the parent Kptfile.
+		_ = revertFunc()
```

Comment:
This means the parent Kptfile is updated, get failed and revert failed. It is highly unlikely that updateParentKptfile() is success and revertFunc() throws error. 

This need not go in now but just a thought. How about combining 2 error messages so that it is more clear to the user as to why their parent Kptfile is modified?

e.g. unable to fetch the package foo due to error blah, failed to revert the parent Kptfile, please clean the subpackages section of parent section